lib/tools.js

```diff
@@ -220,7 +220,9 @@ function prettifierMetaWrapper (pretty, dest) {
 
       var time = this.lastTime
 
-      if (time.match(/^\d+/)) {
+      if (time == null) {
+        time = JSON.parse(chunk).time
```

Comment:
I think this is actually a bug in pino-multi-stream. It should set `lastTime`, See https://github.com/pinojs/pino-multi-stream/blob/28443984d06c43ffaf332165c1bd5395936bda25/multistream.js#L57.

Reply:
@mcollina yeap, that is right.